mlrun/db/base.py

```diff
@@ -123,3 +124,34 @@ def list_projects(self):
 
     def list_artifact_tags(self, project):
         return []
+
```

Comment:
You're missing the type annotations here, please add

Reply:
👍